src/modules/map/MapModule.vue

```diff
@@ -44,7 +44,7 @@ const isPhoneMode = computed(() => uiMode.value === UIModes.PHONE)
             <teleport to="#toolbox-compass-button">
                 <OpenLayersCompassButton />
             </teleport>
-            <CompareSlider />
+            <CompareSlider v-if="isCompareSliderActive" />
```

Comment:
this `isCompareSliderActive` isn't defined above, maybe you might want to add the store binding too